modules/packages/ConcurrentMap.chpl

```diff
@@ -1062,7 +1062,7 @@ module ConcurrentMap {
 
       :arg ch: A channel to write to.
     */
-    proc writeThis(f) throws {
+    proc writeThis(ch) throws {
```

Comment:
I investigated a bit more this issue and in fact this type is already written in some tests, such as `test/library/packages/ConcurrentMap/testAddSet.chpl` or `test/library/packages/ConcurrentMap/testClear.chpl`. However, these tests are skipped since they are future tests, meaning that this issue is not detected.

Reply:
`start_test ConcurrentMap --futures` revealed the following warnings/errors for all tests:
```
> $CHPL_HOME/modules/packages/ConcurrentMap.chpl:84: warning: the RandomStream class is deprecated; please use the randomStream record instead
> $CHPL_HOME/modules/packages/ConcurrentMap.chpl:287: warning: the RandomStream class is deprecated; please use the randomStream record instead
> $CHPL_HOME/modules/packages/ConcurrentMap.chpl:216: warning: missing '(?)' after a generic parent class
> $CHPL_HOME/modules/packages/ConcurrentMap.chpl:238: warning: missing '(?)' after a generic parent class
> $CHPL_HOME/modules/packages/ConcurrentMap.chpl:279: warning: missing '(?)' after a generic parent class
> $CHPL_HOME/modules/packages/ConcurrentMap.chpl:1065: In method 'writeThis':
> $CHPL_HOME/modules/packages/ConcurrentMap.chpl:1066: error: 'ch' undeclared (first use this function)
```
I could propose a separate PR to fix this.

Reply:
It also seems to me that it is not specific to ConcurrentMap. Some future tests in EpochManager also report unexpected warnings.

Reply:
Ah, good point.  I think it's likely we run those futures nightly, but we probably don't have a .bad for them and so didn't notice that they started failing with these warnings.  I'd imagine we'd otherwise have noticed when we tried to remove the deprecation warning.  I wouldn't feel the need to fix the EpochManager ones, but it'd be helpful to open a low priority issue noting the failure so someone on our team can get to it